common/src/main/java/com/discordsrv/common/core/profile/DiscordProfileData.java

```diff
@@ -25,14 +25,16 @@ public class DiscordProfileData {
 
     private final long userId;
     private final Set<String> grantedRewards;
+    private final Set<String> pendingRewards;
 
     public DiscordProfileData(long userId) {
-        this(userId, new HashSet<>());
+        this(userId, new HashSet<>(), new HashSet<>());
     }
 
-    public DiscordProfileData(long userId, Set<String> grantedRewards) {
+    public DiscordProfileData(long userId, Set<String> grantedRewards, Set<String> pendingRewards) {
```

Comment:
Let's make a type for this (`Reward`, with identifier and pending as fields) instead of two separate sets